examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -32,7 +32,7 @@ import {
   CONTRAST_MAP_STYLES,
   INITIAL_TILE_COLOR_MODE,
   INITIAL_BOUNDING_VOLUME_COLOR_MODE,
-  BOUNDING_VOLUME_TYPE
+  BOUNDING_SPHERE
```

Comment:
For consistency, shouldn't this be `INITIAL_BOUNDING_VOLUME_TYPE`?

Reply:
Done